docs/layers/base-layer.md

```diff
@@ -147,25 +129,26 @@ that are not Web Mercator projected longitudes/latitudes.
 
 ##### `projectionMode` (Number, optional)
 
-Specifies how layer positions and offsets should be interpreted.
+Specifies how layer positions and offsets should be geographically interpreted.
 
 The default is to interpret positions as latitude and longitude, however it
-is also possible to interpret positions as meter offsets (measured from the
-`projectionCenter` prop).
+is also possible to interpret positions as meter offsets on top of a projection
+center specified by the `projectionCenter` prop.
 
 See the article on Coordinate Systems for details.
 
 ##### `projectionCenter` ([Number, Number], optional)
 
-Required when the `projectionMode` is in meter offsets.
+Required when the `projectionMode` is set to `COORDINATE_SYSTEM.METER_OFFSETS`.
 
 Specifies a longitude and a latitude from which meter offsets are calculated.
 
 See the article on Coordinate Systems for details
 
+TODO: This section needs to be rewritten
 ##### `modelMatrix` (Number[16], optional)
 
-An optional 4x4 matrix that is premultiplied into the affine projection
+An optional 4x4 matrix that is multiplied into the affine projection
 matrices used by shader `project` GLSL function and the
 Viewport's `project` and `unproject` JavaScript function.
 
```

Comment:
Thanks for the input. Yes, I'd like to write something similar but decided to do it later. I added a TODO for now.